### ISSUE TITLE
Fix circuits for Python >= 3.7

### ISSUE THREAD

**Issue by spaceone** [circuits/circuits#276]
```
Traceback (most recent call last):
  File "/home/travis/build/circuits/circuits/circuits/core/manager.py", line 832, in processTask
    value = next(task)
RuntimeError: generator raised StopIteration```

There is a change in Python 3.7 that a RuntimeError is raised if a generator raises StopIteration.
generator should instead just return.

**Comment by ri0t** [comment]
I replaced those raises with return here and the test suite seems happy. Can prepare PR.

**Comment by spaceone** [comment]
> I replaced those raises with return here and the test suite seems happy. Can prepare PR.

There is already a PR for this: https://github.com/circuits/circuits/pull/275/commits

**Comment by spaceone** [comment]
Fixed in #275
